itest/lnd_route_blinding_test.go

```diff
@@ -1461,3 +1461,105 @@ func testBlindedPaymentHTLCReForward(ht *lntest.HarnessTest) {
 		require.Fail(ht, "timeout waiting for sending payment")
 	}
 }
+
+// testGivenBlindedRouteInvoices tests lnd's ability to:
```

Comment:
nit: comment doesnt match function name. 

Also, I think `testPartiallySpecifiedBlindedPath`is a bit clearer (since "restriction" could refer to many of the restrictions we have available for blinded paths)